<launch>

  <node name="mav_coverage_planning"
    pkg="mav_coverage_planning"
    type="sweep_planner_node"
    output="screen"
    clear_params="true" >

    <!-- Basic and control parameter -->
    <rosparam command="load" file="$(find mav_coverage_planning)/cfg/base_planner.yaml" />
    <!-- Polygon, altitude, and world frame ID -->
    <rosparam command="load" file="$(find mav_coverage_planning)/cfg/example_polygon.yaml" />
    <!-- Sweep planner specific parameter -->
    <param name="fov_camera_rad" value="1.57" />
    <param name="min_view_overlap" value="0.2" />
    <param name="solver" value="2" /> <!-- [0: exact offline, 1: exact online, 2: approximate MA] -->
  </node>

</launch>
